src/Modules/OrchardCore.Commerce/Endpoints/Extentions/ConvertLocalizedHtmlString.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.AspNetCore.Mvc.Localization;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OrchardCore.Commerce.Endpoints.Extensions;
+public static class ConvertLocalizedHtmlString
+{
+    public static string ConvertLocalizedHtmlStringList(this IReadOnlyList<LocalizedHtmlString> errors)
+    {
+        var list = (from item in errors
+                    let strItem = item.Value
+                    select strItem).ToList();
+        string strError = string.Join('\n', list);
+        return strError;
```

Comment:
Prefer `System.Linq` extension methods over LINQ expressions. Also don't use Hungarian Notation in variable names.

The `ToList` conversion is also unnecessary so this can be simplified into
`return string.Join('\n', errors.Select(error => error.Value))`.